### PR TITLE
Added Offset Getters and Setters

### LINKED ISSUE #187: `Canvas.offset` for retrieving the current offset
It's impossible to retrieve the current canvas offset, making it hard to temporarily reset it.

We already have `Canvas.offset()` and `Canvas.offset(x, y)` for setting the offset, but no way to retrieve it. Perhaps a `Canvas.offset` getter (with a reciprocal setter) could provide the result as a `Vector`.

### PR DESCRIPTION
**Pull request by TheKing0x9**
This pulls adds 
- Canvas.offsetX - returns X offset
- Canvas.offsetY - returns Y offset
- Canvas.offset getter - returns Vector of offset X and offset Y
- Canvas.offset setter - sets Canvas offset

Closes #187

### CODE REVIEW

File: docs/modules/graphics.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ This is the name of the default font used for `Canvas.print(str, x, y, color)`.
 This is the height of the canvas/viewport, in pixels.
 #### `static width: Number`
 This is the width of the canvas/viewport, in pixels.
+#### `static offsetX : Number`
```
Comment (avivbeeri) [inline docs/modules/graphics.md:27]: I don't think we should document offsetX and offsetY as part of the public API, unless you can write values to them as well.

Comment (TheKing0x9) [reply docs/modules/graphics.md]: Yep those shouldn't have been there. Fixed it.

File: docs/modules/graphics.md [hunk 2]
```diff
@@ -24,12 +24,8 @@ This is the name of the default font used for `Canvas.print(str, x, y, color)`.
 This is the height of the canvas/viewport, in pixels.
 #### `static width: Number`
 This is the width of the canvas/viewport, in pixels.
-#### `static offsetX : Number`
-This is the X offset of canvas/viewport, in pixels.
-#### `static offsetY : Number`
-This is the Y offset of canvas/viewport, in pixels.
 #### `static offset : Vector`
-A vector of _(Canvas.offsetX, Canvas.offsetY)_ for convenience. You can both get and set this value
+A vector of representing the Canvas offset. You can both get and set this value
```
Comment (avivbeeri) [inline docs/modules/graphics.md:28]: This line should be "A vector representing the canvas offset. You can get and set this value.

**Review by avivbeeri — CHANGES_REQUESTED** [review]

File: docs/modules/graphics.md (hunk 2 above)
Comment (TheKing0x9) [reply docs/modules/graphics.md]: Apologies. It has been fixed

**Comment by avivbeeri** [comment]
Looks good, thank you! Feel free to add your name and contact details to the AUTHORS.md file as well!

**Review by avivbeeri — APPROVED** [review]

**Pull request merged**